samples/cpp/speech_sample/speech_sample.hpp

```diff
@@ -125,8 +125,9 @@ static const char layout_message[] =
 ;
 
 /// @brief message for PWL max error percent
-static const char pwl_max_error_percent_message[] = "Optional. The maximum percent of error for PWL function."
-                                                    "The value must be in <0, 100> range. The default value is 1.0.";
+static const char pwl_approximation_mode_message[] =
```

Comment:
I wonder if constexpr would be ok?

Reply:
I have used the same convention as for other variables in the file.